README.md

```diff
@@ -128,3 +134,65 @@ model = wspc.load_model(model_path)
 X_test = wspc.read_genomes(path_to_genomes)
 predictions = wspc.predict(X_test, model)
 ```
+
+## WSPC input:
+
+WSPC handle different types of input:
+
+1. Input directory with genome *.tab and\or *.txt files:
+
+   *.tab file - Public genomes on PATRIC database are available through a genomes directory. Each genome directory includes a
+   .features.tab file, which provides all genomic features and related information in tab-delimited format, including
+   PGFams information.
+   For features.tab file example, look at the file:
+    https://github.com/shakedna1/wspc_rep/blob/main/Data/Bacpacs/patric_files/1041522.28.PATRIC.features.tab
+
+    *.txt file - Output file of the PATRIC annotation service for new genome. For more detailes on the file and the
+    annotation service, see explanation at the section:
+     "Obtain PATRIC Global Protein Families (PGFams) annotations for new sequenced genome" below.
+
+2. Merged input *.fasta file:
+   A merged file in a fasta format that contains concatenation of the PGFams information, which can be extracted from
+   a *.tab file using the field: pgfam_id and from a *.txt file using the fiels: "pgfam".
+
+   For the merged file exact format, look at the file: https://github.com/shakedna1/wspc_rep/blob/changes_30_7/Data/train_genomes.fasta.
```

Comment:
The path should be 
https://github.com/shakedna1/wspc_rep/blob/main/Data/train_genomes.fasta

Reply:
Yes, my mistake